docs/interacting-with-openshift.md

```diff
@@ -99,48 +99,58 @@ $ minishift logs
+
+### Use-case examples
+
+#### Configuring cross-origin resource sharing
+
+In this use-case example, you update the OpenShift master configuration to change the resource IP
+address in [cross-origin resource sharing](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) (CORS).
+
+By default, OpenShift only allows cross-origin resource requests from the IP address of the
+cluster or from localhost. You specify the IP address in the `corsAllowedOrigins` property of the
+[master configuration](https://docs.openshift.com/enterprise/3.0/admin_guide/master_node_configuration.html#master-configuration-files) (_master-config.yaml_).
+
+To change the property value and allow cross-origin requests from all domains,
+run the following command:
 
 ```
 $  minishift openshift config set --patch '{"corsAllowedOrigins": [".*"]}'
 ```
 
-Per default, the master configuration is targeted, but you can also patch the node configuration
-by specifying the `target` flag. For more details about the
-`set` command refer to its [synopsis](./minishift_openshift_config_set.md).
+#### Changing the OpenShift routing prefix
+
+In this use-case example, you change the OpenShift routing suffix in the master configuration.
+
+If you use a static routing suffix, you can set the `routing-suffix` flag as part of the
+[`start`](./minishift_start.md) command. However, by default Minishift uses a dynamic routing prefix
+based on [nip.io](http://nip.io/).
+
+When the IP address of the VM is apart of the routing suffix, for example
+_192.168.99.103.nip.io_, you can use [xip.io](http://xip.io/) instead of `nip.io`. The `xip.io` suffix
+is based on the same principles and can be used if you experience issues with `nip.io`.
 
-A second use case for the `openshift config` command is the ability to change OpenShift's routing suffix.
-If you use a static routing suffix, you can just specify the `routing-suffix` flag as part of the
-[`start`](./minishift_start.md) command. However, per default Minishift uses a dynamic routing prefix
-based on [nip.io](http://nip.io/). In this case the VM's IP is part of the routing suffix, for example
-_192.168.99.103.nip.io_. There is an alternative to nip.io called [xip.io](http://xip.io/). It is
-based on the same principles. In case you are experiencing issues with nip.io, you can switch to
-xip.io using the `openshift config` command:
+To set the routing suffix to `xip.io`, run the following command:
 
 ```
-$ minishift openshift config set --patch '{"routingConfig": {"subdomain": "192.168.99.103.xip.io"}}'
+$ minishift openshift config set --patch '{"routingConfig": {"subdomain": "<IP-ADDRESS>.xip.io"}}'
 ```
 
-You need to replace the IP in the above command with the IP of your VM, which you
-can obtain via the [`ip`](./minishift_ip.md) command.
+Make sure to replace the IP address in the above example with the IP address of your Minishift VM.
```

Comment:
I only use IP because that's the name of the command (a name that I don't agree with but I have no control over). The best practice is to always spell out "IP address", especially in cases like this one where we're writing about suffixes and prefixes and routing and network elements.